src/Logary.Facade/Facade.fs

```diff
@@ -315,6 +457,113 @@ module internal Formatting =
     formatExn message.fields +
     formatFields matchedFields message.fields
 
+  let literateExceptionColorizer (context: LiterateContext) (ex: exn) =
+    let stackFrameLinePrefix = "   "
+    use exnLines = new System.IO.StringReader(ex.ToString())
+    let rec go lines =
+      match exnLines.ReadLine() with
+      | null -> lines // finished reading
+      | line -> if line.StartsWith(stackFrameLinePrefix) then
+                  go (lines @ [ line, Subtext; Environment.NewLine, Text ])
+                else
+                  go (lines @ [ line, Text; Environment.NewLine, Text ])
+    go []
+
+  let literateLogLevelTokenizer = function
```

Comment:
Maybe best to move this to the `LiterateOptions` like theme? I prefer 3 letter codes, but really it's a personal thing, I find it slightly too hard to notice the colour/level so I think it's worse readability